prelims/Program_Demo.R

```diff
@@ -0,0 +1,166 @@
+# Initialize a list where student data will be contained within vectors.
+# Vectors shall contain the following: ID Num, Name, and Status
+student_list <- list(c(2223175, "Rohit Tank", "Present"))
+
+# The edit_student_attendance allows for the modification of a student's attendance remark
+# based on their assigned student ID. It iterates through the student list and updates the
+# attendance remark for the matching ID. If the student is found, the
+# attendance remark is updated, and the modified list is returned. If the
+# student is not found, the function prints an error message and returns the
+# unchanged list.
+edit_student_attendance <- function(id, new_status) {
+  for (i in 1:length(student_list)) {
+    student <- student_list[[i]]
+    if (student[1] == id) {
+      student[3] <- new_status
+      student_list[[i]] <- student
+      cat("Attendance updated successfully.")
+      return(student_list)
+    }
+  }
+}
+
+
+# Function to add a student to the student_list list data structure.
+# The function accepts an id number and student name and creates a vector
+# using the provided student details. This vector is then added to the
+# student_list list.
+add_student <- function(id, student_name) {
+  student <- c(id, student_name, "Present")
+  student_list[[length(student_list) + 1]] <- student
+  cat("Student added successfully.")
+  return(student_list)
+}
```

Comment:
Please refer to Section II of the company's coding standards under SYNTAX - Curly Braces (Allman) Documentation, and ensure compliance with it. Make sure to use the updated curly brace format consistently across the entire code to meet the modified standard. Thank you.

Reply:
![AllmanFormat](https://github.com/letsgoramen/9480-R-final/assets/148677526/c0044bbf-bd17-4aa6-acdc-8852c0662e3f)